packages/combobox/src/ComboboxContainer.tsx

```diff
@@ -31,7 +31,7 @@ ComboboxContainer.propTypes = {
   hasMessage: PropTypes.bool,
   options: PropTypes.arrayOf(PropTypes.any).isRequired,
   inputValue: PropTypes.string,
-  selectionValue: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
+  selectionValue: PropTypes.any,
```

Comment:
This didn't seem to hold up after the type change, and I didn't find a way to define an accurate statement here using `prop-types`. Open to ideas if this seems like an issue.